src/Centreon/Application/Request/AckRequest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+namespace Centreon\Application\Request;
+
+use Centreon\Domain\Acknowledgement\Acknowledgement;
+use Centreon\Domain\Monitoring\Resource as ResourceEntity;
+
```

Comment:
Short description to know the purpose of this class?

Reply:
Sure, I will add it now :) thanks for the suggestion